### PR TITLE
Avoid warnings and update dependencies to spigot 1.11

### PR DESCRIPTION
**Pull request by Aleksey-Terzi**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	<groupId>vg.civcraft.mc.citadel</groupId>
 	<artifactId>Citadel</artifactId>
 	<packaging>jar</packaging>
-	<version>3.7.02</version>
+	<version>3.7.03</version>
```
Comment (ProgrammerDan) [inline pom.xml]: Go ahead and bump it to 3.8.00 to better differentiate between pre-1.11 and post-1.11

File: src/vg/civcraft/mc/citadel/CitadelConfigManager.java [hunk 2]
```diff
@@ -97,23 +97,21 @@ public static int getMaturationScale(String type){
 	}
 	
 	public static ReinforcementEffect getReinforcementEffect(String type){
-		Effect effect = null;
+		Particle effect = null;
 		if (config.getString("reinforcements." + type + ".effect.type") != null) {
 			try {
-				effect = Effect.valueOf(config.getString("reinforcements." + type + ".effect.type"));
+				String effectName = config.getString("reinforcements." + type + ".effect.type");
+				effect = effectName.equals("FLYING_GLYPH") ? Particle.ENCHANTMENT_TABLE: Particle.valueOf(effectName);
```
Comment (ProgrammerDan) [inline src/vg/civcraft/mc/citadel/CitadelConfigManager.java]: Nice legacy nod

File: src/vg/civcraft/mc/citadel/command/commands/Fortification.java [hunk 3]
```diff
@@ -22,7 +21,6 @@
 import vg.civcraft.mc.namelayer.permission.PermissionType;
 
 public class Fortification extends PlayerCommandMiddle{
-	private ReinforcementManager rm = Citadel.getReinforcementManager();
 	private GroupManager gm = NameAPI.getGroupManager();
```
Comment (ProgrammerDan) [inline src/vg/civcraft/mc/citadel/command/commands/Fortification.java]: is this still used post your change below?

File: src/vg/civcraft/mc/citadel/reinforcementtypes/ReinforcementEffect.java [hunk 4]
```diff
@@ -92,24 +77,16 @@ public int getParticleCount() {
 	}
 
 	/**
-	 * 
-	 * @return the distance from which players will be able to see the effect. 
-	 */
-	public int getViewDistance() {
```
Comment (ProgrammerDan) [inline src/vg/civcraft/mc/citadel/reinforcementtypes/ReinforcementEffect.java]: I don't think it's essential here, but I am curious if a similar limitation to view distance is still possible in practice.

File: src/vg/civcraft/mc/citadel/command/commands/Fortification.java (hunk 3 above)
Comment (Aleksey-Terzi) [inline src/vg/civcraft/mc/citadel/command/commands/Fortification.java]: Yes, it is used in few places in execute() method.

File: src/vg/civcraft/mc/citadel/reinforcementtypes/ReinforcementEffect.java (hunk 4 above)
Comment (Aleksey-Terzi) [inline src/vg/civcraft/mc/citadel/reinforcementtypes/ReinforcementEffect.java]: Starting from 1.10.2 spigot separated sound effects and particle effects.
I've tried to find alternatives for view distance in new method, but looks like it is just not allowed to specify.
Probably view distance somehow related now to some "standard" configuration.

**Pull request merged**